database/mysql/mysql_user.py

```diff
@@ -211,14 +221,17 @@ def user_exists(cursor, user, host):
     count = cursor.fetchone()
     return count[0] > 0
 
-def user_add(cursor, user, host, password, new_priv):
-    cursor.execute("CREATE USER %s@%s IDENTIFIED BY %s", (user,host,password))
+def user_add(cursor, user, host, password, encrypted, new_priv):
+    if not encrypted:
+        cursor.execute("CREATE USER %s@%s IDENTIFIED BY %s", (user,host,password))
+    else:
+        cursor.execute("CREATE USER %s@%s IDENTIFIED BY PASSWORD %s", (user,host,password))
```

Comment:
Some of the core feedback on #1581 and #130 given by @morgo was concerning MySQL 5.7 compatibility. The `PASSWORD` function will be deprecated and the methods for updating hashed passwords have changed. Also, Line 252 is affected by this. This has been accounted for in #1581 

I have tested thoroughly on VMs runnig MySQL 5.5, 5.6 and 5.6 Beta.

The hold up at the moment for #1581 is putting together Integration Tests. The current test suite in not functioning on a clean `devel` build. Looks like @the-lars ran into this as well.